test/python/common.py

```diff
@@ -338,16 +342,22 @@ def _(*args, **kwargs):
             discovered_credentials = discover_credentials()
             if account_name in discovered_credentials.keys():
                 credentials = discovered_credentials[account_name]
+
```

Comment:
Instead of adding this block, can you actually call the function somewhere in `wrapper/credentials/__init__.py:discover_credentials()`? There are several places in that function where `return` happens, so moving things a bit might be needed to ensure that, whenever credentials are returned by that function, they always are in the new style. Tackling it at that level will simplify things and avoid us having to explicitly check for the old ones in some cases (for example, in this block).

Reply:
Ok, this has been addressed.